clients/src/main/java/org/apache/kafka/clients/admin/MemberDescription.java

```diff
@@ -30,21 +30,26 @@ public class MemberDescription {
     private final String host;
     private final MemberAssignment assignment;
     private final Optional<MemberAssignment> targetAssignment;
+    private final Optional<Integer> memberEpoch;
+    private final Optional<Boolean> isClassic;
 
     public MemberDescription(String memberId,
```

Comment:
Here, I suppose that we must introduce a new overload of the constructor with the new arguments in order to not break compatibility of existing usages.

Reply:
Thanks for reminder. Updated it.

Reply:
Updated both PR and KIP. Thanks for the suggestion.